### ISSUE TITLE
Snaps don't work with Yaru-dark.

### ISSUE THREAD

**Issue by snydox** [ubuntu/yaru#1352]
I just performed a clean installation of Ubuntu 19.04 Disco Dingo on a blank disk. Once the installation was done, I went ahead and installed Gnome Tweaks, and then I selected Yaru-dark. When I did that, all my snaps became Vanilla Gnome. Is there a way to fix this, and make all my snaps dark? When I go back to regular Yaru, the apps look normal.

I recorded a quick video for you to have a better idea: https://streamable.com/vtxje

PS: I like Darkness.

**Comment by Paz-it** [comment]
@snydox in order to have your snap application to get Yaru-dark theme you need to install gtk-common-themes like this:  `snap install gtk-common-themes`

**Comment by clobrano** [comment]
@Paz-it is right, but it should come installed automatically.
@snydox, could you try to uninstall and reinstall the same snaps?

**Comment by brogan89** [comment]
Hey I'm also having a similar issue, but instead the apps almost disappear other than the text. I'm using  the Communitheme snap package on Ubuntu 18.04 though (but like to feel like im in 19.04 xD). It seems related. 

Heres video https://broganking.online/nextcloud/index.php/s/7JHSRb9W99TSrPG

I tried steps:
made sure i had `gtk-common-themes` installed (already was)
remove `communitheme`
reboot
install `communitheme`
reboot

issue still remains unfortunately.

**Comment by clobrano** [comment]
What about reinstalling the application snap? Not the communitheme one

**Comment by brogan89** [comment]
@clobrano do you mean `snapd`? just to be sure. I reinstalled that and no change.

**Comment by clobrano** [comment]
I meant reinstalling apps like calculator, but I did it myself and does not fix it anymore (in the past it worked)

**Comment by brogan89** [comment]
Sorry I forgot to mention that in my initial post. I did try reinstalling calculator and system monitor but didn't fix.

**Comment by Dragon8oy** [comment]
Hi, bit late but I had this issue too on 19.04 and after reporting it to the gnome-calculator snap or launchpad page, I was led to gtk-common-themes to file a report. Apparently it was patched (April / Early May) and recently I have noticed the patch on my desktop. It was an issue with yaru-dark being omitted from the themes which was fixed, so not Yaru's fault.

So it's fixed now and apparently nothing was wrong here, at least I think.
